deploy/podman/flightctl-alert-exporter/flightctl-alert-exporter-config/config.yaml

```
@@ -0,0 +1,15 @@
+  port: 5432
+  name: flightctl
+
+service: {}
+
+kv:
+  hostname: localhost
+  port: 6379
+
+alertmanager:
+  hostname: localhost
+  port: 9093 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)